Makefile

```diff
@@ -28,13 +28,13 @@ GRPID   := $(shell id -g)
 .PHONY: install-hooks
 install-hooks:
 	@echo "===> Copying Antrea git hooks to local <==="
-	install hack/git_client_side_hooks/pre-commit .git/hooks/
+	install hack/git_client_side_hooks/* .git/hooks/
 
 .PHONY: uninstall-hooks
 uninstall-hooks:
 	@echo "===> Removing Antrea git hooks from local <==="
-	rm .git/hooks/pre-commit
-
+	rm .git/hooks/*
```

Comment:
nit: maybe we should just delete the files we installed

we could set a Makefile variable like this: `GIT_HOOKS          := $(shell find hack/git_client_side_hooks -type f -print)`. And then use it to define both Makefile targets. What do you think?